### PR TITLE
SAK-44611 Forums: Rubric icon does not appear from ELFinder links

### PR DESCRIPTION
**Pull request by josecebe**

### CODE REVIEW

**Review by ern — APPROVED** [review]

File: webcomponents/tool/src/main/frontend/js/rubrics/rubrics-element.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ class RubricsElement extends SakaiElement {
 
     super();
 
-    this.locale = window.portal ? window.portal.locale : window.top.portal.locale;
+    this.locale = window.portal ? window.portal.locale : window.top.portal ? window.top.portal.locale : window.top.sakai.locale.userLocale.replace("_","-");
```
Comment (adrianfish) [inline webcomponents/tool/src/main/frontend/js/rubrics/rubrics-element.js:9]: You can simplify this with:
this.locale = (window.top?.portal?.locale || window.top?.sakai?.locale?.userLocale || "en-US").replace("_", "-");

**Review by adrianfish — CHANGES_REQUESTED** [review]
I've suggested a terser and safer line of js, using optional chaining.

File: webcomponents/tool/src/main/frontend/js/rubrics/rubrics-element.js (hunk 1 above)
Comment (adrianfish) [reply webcomponents/tool/src/main/frontend/js/rubrics/rubrics-element.js:9]: What that line means is:
if we have window.top.portal.locale, use it, else, if we have window.top.sakai.locale, use that, else use en-US. Then replace _ with - on whatever we end with.

Comment (josecebe) [reply webcomponents/tool/src/main/frontend/js/rubrics/rubrics-element.js]: Thanks for the suggestion @adrianfish , I've just uploaded the changes.

Comment (ottenhoff) [reply webcomponents/tool/src/main/frontend/js/rubrics/rubrics-element.js]: this is probably out of scope for this JIRA, but there is no guarantee we are window.top!!!!!

**Review by adrianfish — APPROVED** [review]

**Pull request merged**